content/english/java-basics/activity-1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Print Statements and Comments"
+description: "Introduce println() methods and comments in Java."
+date: 2020-07-08T00:00:00Z
+---
+
+### 1. Let the Program Speak! 
+
+Let’s try to print `Hello World` out by typing `System.out.print("Hello World");` 
+
+As you might have guessed, the program prints out whatever you typed in the parenthesis. And, each line of code ends with `;`.
+
+After, add more statements to print numbers by putting numbers in the parenthesis (i.e. `System.out.print(2020)`) or a symbol by putting one symbol between 2 single quotes `'` (i.e. `System.out.print('@')`).
+
+<iframe height="600px" width="100%" src="https://repl.it/@nuevofoundation/HelloWorld?lite=true#Main.java" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
```

Comment:
Can you append JavaBasics (or something similar, I know it's probably in a Java Basics folder) to the Repl.it name so that it's easier to identify from just looking at the URL?